### DISCUSSION TITLE
About validating json5

### DISCUSSION THREAD

**Discussion by l0g4n1337** [marioparaschiv/nitro-sniper#360] [Q&A] [answered]
Hi, on replit I have no problems, I wanted to try it locally but I get this error "Invalid settings, please visit the FAQ for more information." .. I checked the FAQ but I don't understand what exactly I need to do for validating.
I hope someone can enlighten me!

EDIT: nvm solved!

**Comment by FrazixOFC** [comment]
can you help me with the json5 valid thing please

**Reply by Craftzman7** [reply]
Read the wiki.

**Reply by ghost** [reply]
I don't completely understand everything on the wiki that talks about Json5 validation. Could you please explain?

**Reply by Craftzman7** [reply]
What exactly do you not understand?

**Comment by marioparaschiv** [comment, marked as answer]
Please use Use https://jsonformatter.org/json5-validator to validate your config.
